submissions/yhorodechna/html-css-popup/index.html

```diff
@@ -0,0 +1,137 @@
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/compass.png" alt="" />
+                                <span class="subtitle">Compass</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/design.png" alt="" />
+                                <span class="subtitle">Design</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/download.png" alt="" />
+                                <span class="subtitle">Download</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/favorite.png" alt="" />
+                                <span class="subtitle">Favorite</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/location.png" alt="" />
+                                <span class="subtitle">Location</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/mail.png" alt="" />
+                                <span class="subtitle">Mail</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/message.png" alt="" />
+                                <span class="subtitle">Message</span>
+                            </a>
+                        </li>
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/phone.png" alt="" />
+                                <span class="subtitle">Phone</span>
+                            </a>
+                        </li>
+                        <li class="popup__more more">
+                            <input class="more__input" type="checkbox" id="more__input" />
+                            <label class="more__label" for="more__input"><span class="subtitle more__btn">
+                                </span></label>
```

Comment:
When the second part of the popup is open your button looks not ok) (not 100% width) also on hover and focus button background overlaps the bottom border of the popup. Also don't forget about border-radius:button more doesn't have it and popup has)
![image](https://user-images.githubusercontent.com/16196199/184407488-115fa39a-408d-46f7-9196-11329b4b01a1.png)

Reply:
It was my big mistake... 
Fixed.